pkgs/applications/audio/buzztrax/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, lib, fetchFromGitHub, wrapGAppsHook, makeFontsConf
+, automake, autoconf, gtk-doc, intltool, libtool, pkgconfig, yelp-tools, itstool
+, gst_all_1
+, glib, libgsf, libxml2
+, clutter-gtk, gtk2
+, orc, fluidsynth
+}:
+
+stdenv.mkDerivation {
+
+  name = "buzztrax-git";
+
+  src = fetchFromGitHub {
+    owner = "Buzztrax";
+    repo = "buzztrax";
+    rev = "833287c6a06bddc922cd346d6f0fcec7a882aee5";
+    sha256 = "sha256-iI6m+zBWDDBjmeuU9Nm4aIbEKfaPe36APPktdjznQpU=";
+  };
+
+  postUnpack = ''
+    touch $sourceRoot/AUTHORS
+  '';
+
+  NIX_CFLAGS_COMPILE = [
+    ## 'g_memdup' is deprecated: Use 'g_memdup2' instead
+    "-Wno-error=deprecated-declarations"
+  ];
+
+  # FONTCONFIG_FILE = makeFontsConf {
+  #   fontDirectories = [ ];
+  # };
+
+  configureScript = "./autogen.sh";
+  configureFlags = [
+    "--enable-debug=no"
+  ];
+
+  nativeBuildInputs = [
+    automake autoconf gtk-doc intltool libtool pkgconfig
+    wrapGAppsHook yelp-tools itstool
+  ];
+
+  buildInputs = [
+
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    glib libgsf libxml2
+    clutter-gtk gtk2
+
+    ## Optional
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+    orc
+    fluidsynth
+  ];
+
+  meta = with lib; {
+    description = "Buzztrax is a modular music composer for Linux.";
+    homepage = "https://www.buzztrax.org/";
+    license = licenses.lgpl2;
```

Comment:
```suggestion
    license = licenses.lgpl21Plus;
```

Reply:
`lgpl21` is deprecated in nixpkgs. The default substitute is `lgpl21Plus`, unless `lgpl21Only` is made explicit by upstream.